conf/cb-event-forwarder.example.ini

```diff
@@ -112,7 +112,7 @@ output_format=json
 #
 
 # default to /var/cb/data/event_bridge_output.json
-outfile=/var/cb/data/event_bridge_output.json
+outfile=/var/cb/data/event_bridge_output.json.gz
```

Comment:
Revert the outfile back to the non-gz version, so the default non-compressed case writes to a file without a gz extension